src/maus/reader/mig_xml_reader.py

```diff
@@ -70,6 +68,21 @@ def __init__(self, init_param: Union[str, Path]):
         self._sanitized_tree: etree.ElementTree = etree.ElementTree(self._sanitized_root)
         make_tree_names_comparable(self._sanitized_tree)
 
+    def _unpack_virtual_groups(self) -> None:
+        """
+        unpacks groups with @meta.virtual="true".
+        Any <root><group meta.virtual="true"><foo/><bar/></group></root> will be unpacked and modify the root to
+        <root><foo/><bar/></root>
+        :return:
+        """
+        elements_that_have_foo_groups = self._original_root.xpath('//class[@meta.virtual="true"]/parent::*')
+        for element_that_has_foo_groups in elements_that_have_foo_groups:
```

Comment:
holy das naming hier ist hart. Mitten im Wort `has` vs `have`.
Kannst du das noch verbessern?

Reply:
ne, es ist doch richtig:
* EIN element that ha**s** foo groups
* VIELE element**s** that ha**ve** foo groups

oder übersehe ich was?

Reply:
Grammatikalisch stimmt das, es ist nur schwer den Unterschied zu finden bei so einem langen Namen und wenn der Unterschied mitten im Namen ist.

Reply:
Vielleicht so :`elements_with_foo_groups` und `one_element_with_foo_groups`?